### PR TITLE
feat: testing new static search

### PR DESCRIPTION
**Pull request by RobPasMue**
TEST PR

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 import time
 
 from ansys_sphinx_theme import (
+    ALL_NODES,
+    PARAGRAPHS,
+    TITLE,
```
Comment (jorgepiloto) [inline doc/source/conf.py:12]: Minor suggestion. Since we are naming nodes in plural, let's do the same of this one:

```suggestion
    TITLES,
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]

File: doc/source/conf.py [hunk 2]
```diff
@@ -267,6 +269,11 @@ def intersphinx_pyansys_geometry(switcher_version: str):
     "design.fa-build",
 ]
 
+index_patterns = {
+    "api/": ALL_NODES,
+    "examples/": TITLE + PARAGRAPHS,
```
Comment (Revathyvenugopal162) [inline doc/source/conf.py:274]:
```suggestion
    "examples/": TITLES + PARAGRAPHS,
```

File: doc/source/conf.py [hunk 3]
```diff
@@ -7,6 +7,8 @@
 import time
 
 from ansys_sphinx_theme import (
+    ALL_NODES,
+    PARAGRAPHS,
```
Comment (Revathyvenugopal162) [inline doc/source/conf.py:11]:
```suggestion
    PARAGRAPHS,
    TITLES,
```

**Comment by jorgepiloto** [comment]
I can see the search is indexing new nodes. Despite this, the anchors are not correct @Revathyvenugopal162:

Consider the <http://localhost:8000/api/ansys/geometry/core/sketch/arc/Arc.html> page. If searching for `from_three_points`, the breadcrumbs are:

```console
Arc > Overview        # This one is valid
Arc > Method detail   # This should point to the right anchor
```

However, they should be:

```console
Arc > Overview
Arc > Method detail > from_three_point
```

**Comment by Revathyvenugopal162** [comment]
> I can see the search is indexing new nodes. Despite this, the anchors are not correct @Revathyvenugopal162:
> 
> Consider the https://geometry.docs.pyansys.com/version/stable/api/ansys/geometry/core/sketch/arc/Arc.html page. If searching for `from_three_points`, the breadcrumbs are:
> 
> ```
> Arc > Overview        # This one is valid
> Arc > Method detail   # This should point to the right anchor
> ```
> 
> However, they should be:
> 
> ```
> Arc > Overview
> Arc > Method detail > from_three_point
> ```

improved the header and the indexing for api references
image

**Comment by jorgepiloto** [comment]
I edited your comment, @Revathyvenugopal162 by attaching the right image. This is working as expected.

**Comment by RobPasMue** [comment]
Hi both! What's the status here?

**Comment by Revathyvenugopal162** [comment]
> Hi both! What's the status here?

It is working as expected and we will release it with ansys-sphinx-theme 1.2

**Comment by RobPasMue** [comment]
@Revathyvenugopal162 - now that 1.2 is out, can we get this merged?

**Comment by Revathyvenugopal162** [comment]
# closing with release of ansys-spshinx-theme v1.2.0

**Pull request closed without merging**